docs/simulation/particle_tracker/index.rst

```diff
@@ -0,0 +1,19 @@
+.. _particle_tracker_module:
+
+===================================================
+`plasmapy.simulation.particle_tracker`
+===================================================
+
+.. currentmodule:: plasmapy.simulation.particle_tracker
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Sub-Packages & Modules
+
+API
+---
+
+.. automodapi:: plasmapy.simulation.particle_tracker
```

Comment:
```suggestion
=======================================================
`plasmapy.simulation.particle_tracker.particle_tracker`
=======================================================

.. currentmodule:: plasmapy.simulation.particle_tracker.particle_tracker


.. toctree::
   :maxdepth: 1
   :caption: Sub-Packages & Modules

API
---

.. automodapi:: plasmapy.simulation.particle_tracker.particle_tracker
```
I'm not quite sure what's happening with the doc build error, but I think it's associated with this file.  In Python, generally a `.py` file is called a module, while a directory corresponds to a subpackage.  My _guess_ is that the `currentmodule` directive is pointing to a subpackage instead of a module, which is why I changed the namespaces to refer to a module.  I'm not sure about this, however. (The `reference target not found` error is a pain to deal with since it can refer to so many different things!)

Another possibility would be to have `particle_tracker.rst` be in the `simulation` folder instead of `simulation/particle_tracker/index.rst`, and have the individual module be a stub file.

Reply:
Okay wait so I think I got everything working. Is the new structure of the documentation how you wanted it? (i.e. stubs for the individual modules and then one single index.rst for the subpackage) The way I structured the index.rst for the particle_tracker module is based off the index.rst for the index.rst of the parent subpackage-- plasmapy.simulation

Reply:
Thanks!  The structure of the documentation looks good to me!